src/odemis/acq/stitching/_weaver.py

```diff
@@ -43,66 +44,233 @@ def collageWeaver(tiles, border=None):
     return (2D DataArray): same dtype as the tiles, with shape corresponding to
       the bounding box.
     """
+    
+    def __init__(self):
+        self.tiles = []
+
+    def addTile(self,tile):
+        """
+        tile (2D DataArray): the image must have at least MD_POS and
+        MD_PIXEL_SIZE metadata. All provided tiles should have the same dtype.
+        """
+        
+        self.tiles.append(tile)
+    
+       
+    def getFullImage(self):
+        """
+        return (2D DataArray): same dtype as the tiles, with shape corresponding to the bounding box. 
+        """
+             
+        # Sort the tiles by time, to avoid random order in "Z", and high-light the
+        # acquisition order.
+        tiles = sorted(self.tiles, key=lambda t: t.metadata.get(model.MD_ACQ_DATE, 0))
```

Comment:
Now that there is .addTile, it doesn't make much sense to order them by date afterwards. => drop this line, and check if some of the callers should get this "hack" now.